index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+        body{
+            background-color: rgba(104, 133, 80, 0.849);
+            background-image: url("https://wallpaperaccess.com/full/2041777.jpg");
+            text-emphasis-color: red;
+        }
+        #title{
+            text-align: center;
+            font-size: 40px;
+        }
+        #survey-form{
+            text-align: center;
+        }
+        #description{
+            text-align: center;
+            font-size: 20px;
+        }
+    </style>
```

Comment:
For best practices, please use a separated file for CSS styling and than link it to the HTML file.